docs/extensions/custom-auth/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+    "label": "Signature Authenticators",
```

Comment:
```suggestion
    "label": "Authenticators",
```

aligns with title in extensions overview

Reply:
The other labels do not align with the extensions overview.